src/main/jflex/flowg.flex

```diff
@@ -28,6 +28,9 @@ import java.math.BigDecimal;
 
 %debug
 
+
+%states Default, GCodePreState, GcodeFunctionState
```

Comment:
The Default (initial state) is called YYINITIAL in jflex and is implicitly defined